api/graphql/merchants/merchant_mutations.py

```diff
@@ -0,0 +1,73 @@
+import graphene
+from django.conf import settings
+from graphene import relay
+from graphene_permissions.mixins import AuthMutation
+from sentry_sdk import capture_message
+
+from merchants.models import PaymentOrder, PaymentStatus
+from merchants.verkkokauppa.payment.exceptions import GetPaymentError
+from merchants.verkkokauppa.payment.requests import get_payment
+from reservations.email_utils import send_confirmation_email
+from reservations.models import STATE_CHOICES
+
+from ..validation_errors import ValidationErrorCodes, ValidationErrorWithCode
+
+
+class RefreshOrderMutation(relay.ClientIDMutation, AuthMutation):
```

Comment:
Do we need some permissions to perform this operations? This is ``AuthMutation`` but no ``permission_classes`` are defined 🕵️

Reply:
The caller cannot really "do" anything with this since this is just for triggering an action that should happen behind the scenes automatically. So there isn't big harm that can be done by doing this. I'll double check it.

Reply:
Oh and if I'm not mistaken, this is not called via admin UI but via public UI after customer paid the order. I have a feeling we can't have any roles, but perhaps we could check that the caller is the same as the owner of the related reservation 🤔

Reply:
There is now a permission checks in place. I couldn't figure out an easy way to do this with existing permission checks, so I created a new `can_refresh_order` helper function.

Reply:
We could add Permission class for this which uses the `can_refresh_order`

Reply:
Done! 👍